dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/AliasHelper.java

```diff
@@ -96,4 +96,8 @@ public static WorkflowVersion addWorkflowVersionAliasesAndCheck(AuthenticatedRes
         newAliases.forEach(alias -> workflowVersion.getAliases().put(alias, new Alias()));
         return workflowVersion;
     }
+
+    public static String createWorkflowVersionAliasUrl(String dockstoreUrl, Workflow workflow, String alias) {
+        return "%s/aliases/%s-versions/%s".formatted(dockstoreUrl, workflow.getEntryTypeMetadata().getTerm(), alias);
```

Comment:
In the web service, t looks like there is only a path for `...workflow-versions...`, [here](https://github.com/dockstore/dockstore/blob/develop/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasResource.java#L74).

Will the UI convert `notebook-versions` to `workflow-versions` when it invokes the Dockstore API?

Trying to make sure I understand correctly.

Reply:
Yes, the URL created by `createWorkflowVersionAliasUrl` references our site, where UI code retrieves information about the alias via the API call you mention, and then redirects.  The latter magic happens in `aliases/aliases.component.ts` and related.